### DISCUSSION TITLE
arc.tables() - pass in settings / credentials

### DISCUSSION THREAD

**Discussion by kenhorn** [architect/architect#1287] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
I'd like to use arc.tables() to access dynamo/documentclient, however, I want to do this via an assumed role. 

**Describe the solution you'd like**
Ideally arc.tables(settings) , which would be passed through to new AWS.DynamoDB(settings), and on to the DocumentClient wrapper.

**Describe alternatives you've considered**
Currently just using DocumentClient directly, via above settings.

**Comment by ryanblock** [comment]
What's the use case for this? It's not a bad idea, but this Architect plugin may be what you're looking for if the use case is to talk to DynamoDB tables outside of a given Architect app: https://github.com/architect/plugin-external-tables

**Reply by kenhorn** [reply]
My use is to allow architect to define the table names, but be able to adjust other options to the constructor - specifically the credentials so that the connection can be for a specific role/identity (I'm using per tenant credentials, vs the lambda's own role) 

// eg currently opts contains aws credentials
  opts.endpoint = data._db.endpoint.href;
  var db   = new AWS.DynamoDB(opts);
  var doc  = new AWS.DynamoDB.DocumentClient({service:db});
